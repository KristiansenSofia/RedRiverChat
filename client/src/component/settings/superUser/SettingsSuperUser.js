import React, { Component } from 'react';

// Import styles. settingsSuperUserStyles for all imported components with a style attributes and CSS-file for classNames and id.
import {settingsSuperUserStyles} from "../../../styles/SettingsStyles";
import '../../../styles/Styles.css'

/**
 *  SettingsSuperUser-component.
 *
 *  @author Jimmy
 */

class SettingsSuperUser extends Component {
    render() {
        return (
            <div className="SettingsSuperUser">
                <p>Superuser settings</p>
            </div>
        );
    }
}

export default SettingsSuperUser;