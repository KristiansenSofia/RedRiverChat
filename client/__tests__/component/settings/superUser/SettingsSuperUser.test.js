import React from 'react';
import SettingsSuperUser from '../../../../src/component/settings/superUser/SettingsSuperUser';
import renderer from 'react-test-renderer';

test('SettingsSuperUser renders without crashing', () => {
  const tree = renderer.create(<SettingsSuperUser />);
  expect(tree.toJSON()).toMatchSnapshot();
});
