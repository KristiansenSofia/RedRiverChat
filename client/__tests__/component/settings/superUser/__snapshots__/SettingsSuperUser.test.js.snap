// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsSuperUser renders without crashing 1`] = `
<div
  className="SettingsSuperUser"
>
  <p>
    Superuser settings
  </p>
</div>
`;
